python/railjson_generator/railjson_generator/rjs_static/switch_types.json

```diff
@@ -76,5 +76,20 @@
                 }
             ]
         }
+    },
+    {
+        "id": "link",
+        "ports": [
+            "source",
+            "destination"
```

Comment:
For consistency I would have named them `src` and `dst`, this seems prone to errors. But it seems like it would be a fairly large change now, it's fine as it is

Reply:
We can rename ports later (when we add primitive node types). We have to think of something better in the meantime.

Reply:
Yes, the next step is a better naming for swicthes, and we'll also rename the ports with primitive node types.